### PR TITLE
Use Ctrl-Shift-/ since Brackets assigned Ctrl-Alt-/

### PR DESCRIPTION
**Pull request by sylvain-galineau**
Since Brackets has assigned your original key binding, I propose using Ctrl-Shift-/. This works with the Sprint 24 build.

### CODE REVIEW

**Comment by peterflynn** [comment]
Hmm... so the shortcut I'm currently using is already taken on Mac, but the one you're suggesting we switch to is already taken on Win.

According to adobe/brackets#2506 says Cmd+Shift+/ is actually taken on Mac too -- reserved by the OS for the 'Help search' feature.  However, that shortcut currently doesn't work in brackets-shell and it will probably be a long time before it does.  So for now, at least, we could use Cmd+Shift+/ on Mac and keep using Ctrl+Alt+/ on Win.

**Comment by sylvain-galineau** [comment]
Doh. Sorry I missed that. It'd be nice to have the same shortcut on both
platforms. I'll try and find some options and get back to you.

**Comment by sylvain-galineau** [comment]
My bad, this was obviously too quick; Ctrl-Shift-? is an OSX shortcut for the Mac Help menu. [1] While keyboard.json assigns Ctrl-Shift-/ on Windows to editor.blockComment.

Keys combined by OSX with Cmd-Alt- are: A, F, I, M, N, T, W, Y, /
Keys combined by OSX with Cmd-Shift are: A, C, D, F, G, H, I, K, N, P, Q, S, T, U, W, Z ?, UP, DN, LEFT, RIGHT, TAB, DEL, `, =, 3, 4
Brackets' use of Ctrl-Alt also blocks: UP, DN,  P, S, /
Brackets' uses Ctrl-Shift: C, D, F, H, O, W, UP, DN, ENTER, /, TAB 
Windows documents [2] using Ctrl-Shift with: TAB, ESC
DELETE seems to be the only official Ctrl-Alt combo in Windows.

Bottom line: Cmd-Shift is busy on the Mac. Brackets/Mac relies on and extends Ctrl-Shift as a result and I suspect it will add more.  

Since this is the equivalent of Cmd-Shift-P in Sublime, why not use that? OSX uses it for print settings which seems a pretty low-use UI in Brackets.

For the safer Cmd-Alt, possible candidates could be :, >,  Z (very close to Ctrl-Alt keys), maybe | ?

[1] http://support.apple.com/kb/ht1343
[2] http://support.microsoft.com/kb/126449

**Comment by njx** [comment]
Cmd-Alt-Shift-/ seems to work fine for me on Mac (though it's obviously a lot of modifiers :))

**Comment by sylvain-galineau** [comment]
Yes, given enough modifiers it will work safely :) Four keys is rather painful for any editor functionality; especially so for something this useful (Cmd-Shift-P and Cmd-P are heavily used in Sublime). I'd prefer to stick to two modifiers. If Cmd-Shift-P is not cool because OSX defines it then how about Cmd-Alt-C? The three keys are close together, C and commands is easy to remember...

**Comment by chrismatheson** [comment]
is there an reason not to continue sublimes lead and use Cmd-P? im on OSX and looking forward to trying this out :)

**Comment by sylvain-galineau** [comment]
Chris, I like using Cmd-P as well. It does override an OSX shortcut though (Print). I have no idea how many people print from a code editor these days; Sublime clearly believes this scenario doesn't matter - Print is not even available in the menus - and doesn't seem to have a problem with it. Their position is a minority one though.

**Comment by peterflynn** [comment]
Ugh, sorry I was AWOL on this thread for so long :disappointed:

I've just pushed a change updating the Mac shortcut to **Ctrl+Cmd+?** for now (Win shortcut unchanged).

I actually think Cmd+Shift+? would be nice, because the "help search" command that Brackets lacks actually does search the menu bar just like this extension.  Unfortunately, because of its special meaning the shortcut doesn't work fully -- my command gets fired properly, but the shortcut isn't listed next to the menu item (even if I use a `displayKey`).

So I went with Cmd+Ctrl.  There a few similar options also free (e.g. Ctrl+Shift+?, Ctrl+Alt+?) but Cmd+Ctrl seemed like the best bet.  It's pretty similar to the physical key positions you use for the Windows shortcut, and for people accustomed to using "help search" for quick menu access, it's fairly similar to the Shift+Cmd finger positions too.

I can see how Cmd+Shift+P would be nice for Sublime users, but I still prefer "?" myself.  There's no consistency amongst other editors with this feature: Ctrl+Shift+A in Webstorm, Ctrl+3 in Eclipse, Ctrl+Space for Quicksilver, etc.  I'm not opposed to having an option for "Sublime-alike" shortcuts... although that's a little tricky to do until Brackets has a [preferences UI](https://trello.com/c/5GwJgKfi/480-preference-dialog).  In the meantime, everyone should feel free to hack your copy locally to change the shortcut to whatever you like -- it's defined at the very bottom of _main.js_.

**Comment by peterflynn** [comment]
Btw @sylvain-galineau Thanks for bringing attention to this issue in the first place (I was spaced out enough to not even realize it was broken on Mac) and for all your research on keyboard shortcut availability!

**Comment by sylvain-galineau** [comment]
That works for me. Thanks!

**Comment by opensas** [comment]
I do second the motion to set it to Ctrl-Shift-P

BTW, any idea why I can't reassign the shortcut using the extension Brackets Key Remapper?
